base/src/main/java/com/mindera/skeletoid/dialogs/AbstractDialogFragment.kt

```diff
@@ -212,7 +206,11 @@ abstract class AbstractDialogFragment : DialogFragment() {
         onClick(CANCELED)
     }
 
-    protected fun onClick(state: DialogState) {
+    protected fun onBackPressed() {
+        onClick(BACK_PRESSED)
+    }
+
+    private fun onClick(state: DialogState) {
```

Comment:
Is there a reason for this to be private instead of protected? We are using it in WR